x-pack/plugins/integration_assistant/server/graphs/kv/prompts.ts

```diff
@@ -0,0 +1,144 @@
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { ChatPromptTemplate } from '@langchain/core/prompts';
+
+export const KV_MAIN_PROMPT = ChatPromptTemplate.fromMessages([
+  [
+    'system',
+    `You are an expert in creating Elasticsearch KV processors.Here is some context for you to reference for your task, read it carefully as you will get questions about it later:
+
+ <context>
+ <samples>
+ {samples}
+ </samples>
+ </context>`,
+  ],
+  [
+    'human',
+    `Looking at the multiple log samples provided in the context, our goal is to create a KV processor that can parse the logs. Ana;yze the logs  to understand their structure, including any key-value pairs, delimiters, and patterns.
+
+ Follow these steps to help improve the KV processor and apply it to each field step by step:
+
+ 1. Based on your analysis of the log samples, identify different key-value pairs and the delimeters that separates them, create a KV processor that can parse log. The processor should correctly handle logs where keys and values are separated by a \`field_split\` and pairs are separated by \`value_split\`.
+ 2. Recognize and properly format different data types such as strings, numbers, and timestamps.
+ 3. Handle quoted values correctly (e.g., error="Insufficient funds").
+ 4. The \`value_split\` is the delimeter regex pattern to use for splitting the key from the value within a key-value pair (e.g., ':' or '=' )
+ 5. The \`field_split\` is the regex pattern to use for splitting key-value pairs in the log. Make sure the regex pattern breaks the log into key-value pairs.
+ 6. Ensure that the KV processor can handle different scenarios, such as: Optional or missing fields in the logs , Varying delimiters between keys and values (e.g., = or :), Complex log structures (e.g., nested key-value pairs or key-value pairs within strings, whitespaces , urls, ipv4 , ipv6 address, mac address etc.,).
+ 7. Use \`trim_key\` for string of characters to trim from extracted keys.
+ 8. Use \`trim_value\` for string of characters to trim from extracted values.
+
+ You ALWAYS follow these guidelines when writing your response:
+ <guidelines>
+ - Use only elasticsearch kv processor.
+ - Do not create any other processors.
+ - Do not add any prefix to the processor.
+ - Do not use the special characters like \`\s\` or \`\\s+\` in the \`field_split\` or \`value_split\` regular expressions.
+ - Do not add brackets (), <>, [] as well as single or double quotes in \`trim_value\`.
+ - Make sure to trim whitespaces in \`trim_key\`. 
+ - Do not respond with anything except the processor as a JSON object enclosed with 3 backticks (\`), see example response below. Use strict JSON response format.
+ </guidelines>
+
+ Example response format:
+
+ <example_response>
+ A: Please find the JSON object below:
+ \`\`\`json
+{ex_answer}
+ \`\`\`
+ </example_response>`,
+  ],
+  ['ai', 'Please find the KV processor below:'],
```

Comment:
Shouldn't this match the example response above? 

ie be 
```suggestion
  ['ai', 'Please find the JSON object below:'],
```